src/cls/IPM/Lifecycle/Module.cls

```diff
@@ -98,17 +98,19 @@ Method %PrepareDeploy(ByRef pParams) As %Status
 
 			Set tName = $Piece(tResource.Name, ".", 1, *-1)
 			Set tExtension = $$$ucase($Piece(tResource.Name, ".", *))
-			continue:tExtension=""
+			if (tExtension = "") || ($Extract(tExtension) = "/") {
+				continue
+			}
 
 			Set tSC = $Case(tExtension,
 				"GBL":##class(%IPM.Utils.Module).AddGlobalMapping(tPackageDeployNamespace, tName, tSource),
 				"INC":##class(%IPM.Utils.Module).AddRoutineMapping(tPackageDeployNamespace, tName, "INC", tSource),
 				"INT":##class(%IPM.Utils.Module).AddRoutineMapping(tPackageDeployNamespace, tName, "INT", tSource),
 				"MAC":##class(%IPM.Utils.Module).AddRoutineMapping(tPackageDeployNamespace, tName, "MAC", tSource),
 				"PKG":##class(%IPM.Utils.Module).AddPackageMapping(tPackageDeployNamespace, tName, tSource),
-				"":##class(%IPM.Utils.Module).AddRoutineMapping(tPackageDeployNamespace, tName, "ALL", tSource))
+				"":##class(%IPM.Utils.Module).AddRoutineMapping(tPackageDeployNamespace, tName, "ALL", tSource),
+				:$$$OK)
```

Comment:
Question: is it our intention to allow other extension names to fail silently?

Reply:
Yes.